docs/changelog.rst

```diff
@@ -37,6 +37,7 @@ New features:
   ``=~``.
   :bug:`4251`
 * :doc:`/plugins/discogs`: Permit appending style to genre
+* :doc:`/plugins/discogs`: State limitation when importing singletons
```

Comment:
Probably no need for a changelog entry when it's just a docs change!

Reply:
Thanks! I wasn't sure about this and saw other `:doc:` entries, so I erred on the side of caution.
I'll remove it.

Reply:
No sure if I used the most optimal git strategy to undo this change (reverted file then committed it), but the GitHub changes section does not show this file. I can investigate other strategies here if needed.